system/lib/compiler-rt/lib/ubsan/ubsan_flags.cpp

```diff
@@ -74,7 +74,7 @@ void InitializeFlags() {
   // Override from user-specified string.
   parser.ParseString(__ubsan_default_options());
   // Override from environment variable.
-#if SANITIZER_EMSCRIPTEN
+#if SANITIZER_EMSCRIPTEN && !defined(__wasm64__)/*FIXME*/
```

Comment:
This type of FIXME might be forgotten as it silently does nothing. How about making it abort if wasm64, and just not considering ubsan supported for now? Or does it work all except for this one thing?

Reply:
It doesn't do "nothing", it does `parser.ParseStringFromEnv("UBSAN_OPTIONS");`, but I have no idea how that differs..

Reply:
Ah, I think without the extra code, it will not properly allocate the buffer here. I'd expect this to fail at runtime then. Is that what you saw?

Reply:
The default codepath here just check the environment variable whereas the emscripten path also check for a JS object.. so I think the net effect of this change is that the `UBSAN_OPTIONS` setting on the Module won't honored for wasm64 (you would need to inject it into the ENV object which is a little more complex).

Reply:
So tests that do this kind of thing in pre-js won't work right away: `Module = {UBSAN_OPTIONS: "print_stacktrace=1"};`

Reply:
Ok, I'll make it abort for now.